pennylane/_qubit_device.py

```diff
@@ -582,19 +583,18 @@ def batch_execute(self, circuits):
             # not start the next computation in the zero state
             self.reset()
 
-            # TODO: Insert control on value here
-            res = self.execute(circuit)
+            # Insert control on value here
```

Comment:
```suggestion
            # TODO: Insert control on value here
```

Reply:
Do you know what is this comment about?

Reply:
Nope :thinking: Checking via `git blame` may show if there's some info in the original PR that added the comment.

Reply:
It was me but I don't remember haha I am removing it